pgmq-extension/sql/pgmq.sql

```diff
@@ -460,14 +460,22 @@ RETURNS TEXT AS $$
     extname = 'pg_partman';
 $$ LANGUAGE SQL;
 
-CREATE FUNCTION pgmq.drop_queue(queue_name TEXT, partitioned BOOLEAN DEFAULT FALSE)
+CREATE FUNCTION pgmq.drop_queue(queue_name TEXT)
```

Comment:
can we overload `drop_queue()` instead so that we can maintain backwards compatibility under semver?

Then we can drop `pgmq.drop_queue(queue_name TEXT, partitioned BOOLEAN DEFAULT FALSE)` when we release v2, which does not yet have a target date.

Reply:
Just to confirm, we shouldn't remove the "partitioned" argument just yet? Just ignore it?

Reply:
What I mean is to add a new function with the signature `pgmq.drop_queue(queue_name TEXT)` (that will look up the queue type) and leave `pgmq.drop_queue(queue_name TEXT, partitioned BOOLEAN DEFAULT FALSE)` the way it currently is.